charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: v1 
+kind: ConfigMap
+metadata:
+  name: {{ include "newrelic.common.naming.fullname" . }}-daemonset-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+data:
+  otel-collector-config.yaml: |
```

Comment:
How the user can change this? Can we expose the config in the values.yaml having all of this as a default?

You can get inspired by the community chart to see how they implemented "set of features"

Reply:
We are currently working on a followup PR that will `expose the config in values.yaml`, which won't be included in this PR